tasks/quality_control/task_consensus_qc.wdl

```diff
@@ -8,7 +8,7 @@ task consensus_qc {
     Int disk_size = 100
   }
   command <<<
-    if [ ~{reference_genome} ] ; then
+    if [ -s "~{reference_genome}" ] ; then
```

Comment:
Oof, good add. Did we get issues with empty reference files?

Reply:
Yup. In one of the iterations of this solution, I had an empty fasta as a placeholder, but that led to failures. This is also just more robust so I kept it in.